highway/client/cosmos.go

```diff
@@ -4,16 +4,22 @@ import (
 	"context"
 
 	"github.com/kataras/golog"
-	"github.com/sonr-io/blockchain/x/registry/types"
+	bt "github.com/sonr-io/blockchain/x/bucket/types"
+	ct "github.com/sonr-io/blockchain/x/channel/types"
+	ot "github.com/sonr-io/blockchain/x/object/types"
+	rt "github.com/sonr-io/blockchain/x/registry/types"
 	"github.com/sonr-io/core/highway/config"
 	"github.com/tendermint/starport/starport/pkg/cosmosclient"
 )
 
 type Cosmos struct {
-	accName  string
-	address  string
-	instance cosmosclient.Client
-	query    types.QueryClient
+	accName string
```

Comment:
Do we need the session context in this struct? I feel we dont as this is on chain but we might want to keep a reference to the session? Curious on what you think